docs/source/horovod.rst

```diff
@@ -1,8 +1,52 @@
 Python Programming
 ==================
 
+Conda environments
+------------------
+To help you work with differents versions of Python and different packages/libraries to avoid compatibility issues.
```

Comment:
If the text is coming from a website, add the proper url here.

Reply:
I figured it out myself!
The steps are a form of mix between docs.conda.io and my personal experience installing it.

Reply:
```suggestion
Conda is a package and environment management system. It allows to easily find and install packages. It is very useful to work with different Python packages/libraries versions, and to avoid compatibility issues.
```

Reply:
```suggestion
Conda is a package and environment management system, allowing to easily find and install packages. It is very useful to work with different Python packages/libraries versions, and to avoid compatibility issues.
```